lib/showoff.rb

```diff
@@ -303,25 +304,25 @@ def process_markdown(name, section, content, opts={:static=>false, :pdf=>false,
       # todo: unit test
       lines = content.split("\n")
       @logger.debug "#{name}: #{lines.length} lines"
-      slides = []
-      slides << (slide = Slide.new)
+      slideset = []
```

Comment:
Oh, I believe you. I've never liked the name collision.

On the other hand, the instance variable *should* be a global `s/@slide/@slideset/` if you want to make that change.